unittests/scans/mobsf/mobsf_3_7_9.json

```diff
@@ -0,0 +1,1288 @@
+       "/usr/lib/libSystem.B.dylib (compatibility version: 1.0.0, current version: 1226.10.1)",
+       "/System/Library/Frameworks/CoreFoundation.framework/CoreFoundation (compatibility version: 150.0.0, current version: 1241.11.0)"
+    ],
+    "files":[
+       "BitbarIOSSample.app/embedded.mobileprovision",
+       "BitbarIOSSample.app/Default@2x.png",
+       "BitbarIOSSample.app/RadioButton-Selected.png",
+       "BitbarIOSSample.app/PkgInfo",
+       "BitbarIOSSample.app/Default-568h@2x.png",
+       "BitbarIOSSample.app/logo_cloud.png",
+       "BitbarIOSSample.app/Default.png",
+       "BitbarIOSSample.app/RadioButton-Unselected.png",
+       "BitbarIOSSample.app/archived-expanded-entitlements.xcent",
+       "BitbarIOSSample.app/bitbar.png",
+       "BitbarIOSSample.app/BitbarIOSSample",
+       "BitbarIOSSample.app/icon.png",
+       "BitbarIOSSample.app/Info.plist",
+       "BitbarIOSSample.app/_CodeSignature/CodeResources",
+       "BitbarIOSSample.app/en.lproj/InfoPlist.strings"
+    ],
+    "urls":[
+       {
+          "urls":[
+             "http://www.apple.com/dtds/propertylist-1.0.dtd"
+          ],
+          "path":"BitbarIOSSample.app/archived-expanded-entitlements.xcent"
+       },
+       {
+          "urls":[
+             "http://www.apple.com/dtds/propertylist-1.0.dtd",
+             "http://www.apple.com/appleca/root.crl0",
+             "https://www.apple.com/appleca/0",
+             "http://www.apple.com/appleca/0m",
+             "http://developer.apple.com/certificationauthority/wwdrca.crl0"
+          ],
+          "path":"BitbarIOSSample.app/BitbarIOSSample"
+       },
+       {
+          "urls":[
+             "http://www.apple.com/dtds/propertylist-1.0.dtd",
+             "https://www.apple.com/appleca/0",
+             "http://developer.apple.com/certificationauthority/wwdrca.crl0",
+             "http://www.apple.com/appleca/0m",
+             "http://www.apple.com/appleca/root.crl0"
+          ],
+          "path":"IPA Strings Dump"
+       }
+    ],
+    "domains":{
+       "www.apple.com":{
+          "bad":"no",
+          "geolocation":{
+             "ip":"92.122.160.209",
+             "country_short":"GB",
+             "country_long":"United Kingdom of Great Britain and Northern Ireland",
+             "region":"England",
+             "city":"Slough",
+             "latitude":"51.509491",
+             "longitude":"-0.595410"
+          },
+          "ofac":false
+       },
+       "developer.apple.com":{
+          "bad":"no",
+          "geolocation":{
+             "ip":"17.253.37.202",
+             "country_short":"GB",
+             "country_long":"United Kingdom of Great Britain and Northern Ireland",
+             "region":"England",
+             "city":"London",
+             "latitude":"51.508530",
+             "longitude":"-0.125740"
+          },
+          "ofac":false
+       }
+    },
+    "emails":[
+       
+    ],
+    "strings":[
```

Comment:
I don't think it's necessary to include quite this much detail in the `strings` array for an example file since none of it is ultimately checked by the test referencing this file

Reply:
Yes, thank you :-)